ansible_image_validation/validate-filtered-images.sh

```diff
@@ -7,7 +7,7 @@ CURR_DIR=$(pwd)
 BASEDIR=$(dirname "$0")
 cd $BASEDIR
 
-echo "" > /var/log/validation.log 
+# echo "" > /var/log/validation.log 
```

Comment:
This line was to clear the previous log. Any specific reason why this is getting commented?

Reply:
yes, this file is manually created, so this script was throwing an error as the file didn't exist. I didn't see this file being used anywhere in the repo. I think you are logging stderr, stdout in this file. Since we moved to the release pipeline, we can remove this file.

Reply:
If this is throwing errors then delete this line instead of commenting it